.travis.yml

```diff
@@ -45,6 +45,19 @@ matrix:
       script:
         - py.test -vv s3fs
 
+    - language: generic
+
+      env:
+        - TEST=PRE-COMMIT
+
+      install:
+        - pip install pre-commit
```

Comment:
If the language is generic, you need to install python - this command is trying to install black into the system python. I would not be against simply adding "black" as the final step of the main tests, which is what dask does.